server/titleflix/src/main/java/io/titleflix/exception/NoGenreFound.java

```diff
@@ -0,0 +1,14 @@
+package io.titleflix.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code=HttpStatus.NOT_FOUND,reason="No Gnere present in the database")
+public class NoGenreFound extends Exception{
```

Comment:
Thanks for the review praveen, I would change most of the exceptions to generic which minimizes the code repetition. Thanks for the feed back.